clang/include/clang/Basic/CodeGenOptions.def

```diff
@@ -189,6 +189,7 @@ CODEGENOPT(OpenCLCorrectlyRoundedDivSqrt, 1, 0) ///< -cl-fp32-correctly-rounded-
 CODEGENOPT(HIPCorrectlyRoundedDivSqrt, 1, 1) ///< -fno-hip-fp32-correctly-rounded-divide-sqrt
 CODEGENOPT(HIPSaveKernelArgName, 1, 0) ///< Set when -fhip-kernel-arg-name is enabled.
 CODEGENOPT(SYCLFp32PrecSqrt, 1, 0) ///< -fsycl-fp32-prec-sqrt
+CODEGENOPT(SYCLUseBuiltinsForIntegration, 1, 0) ///< Set when -fsycl-use-builtins-for-integration is enabled.
```

Comment:
The `SYCLUseBuiltinsForIntegration` option isn't used in this patch. I assume a follow up change will make use of it?

I'm skeptical that this should be a CodeGen option. The behavior I expect the option to govern are the additional compile actions created by `SYCLActionBuilder::getDeviceDependences()` in `clang/lib/Driver/Driver.cpp`.

Reply:
I changed it to a LangOpt.   CodeGenOpt was a thinko.